install/empty_data.php

```diff
@@ -6665,7 +6665,7 @@ public function getEmptyData(): array
             ], [
                 'profiles_id' => self::PROFILE_SUPER_ADMIN,
                 'name' => 'search_config',
-                'rights' => DisplayPreference::PERSONAL | DisplayPreference::GENERAL,
+                'rights' => READ | UPDATE | CREATE | PURGE | DisplayPreference::PERSONAL | DisplayPreference::GENERAL,
```

Comment:
Should we add `READ | UPDATE | CREATE | PURGE` rights to the `DisplayPreference::getRights()` method ?

Right now, if you change the profile configuration for this profile, as these rights are not present in the rights matrix, they will be disabled.

Reply:
Or you can change the `can*` methods to check for the `PERSONAL`/`GENERAL` value.

Reply:
I think we should modify the can* method on display pref and make them use the correct rights that already exist.